src/reducers/globalReducer.js

```diff
@@ -0,0 +1,31 @@
+import initialStore from '../store/initialState';
+import {
+  TRIGGER_AUTHENTICATED,
+  TRIGGER_ERROR,
+} from '../actions/actionTypes';
+
+const { global } = initialStore;
+
+/**
+ * Glabal related state handler
+ *
+ * @export
+ * @param {object} [state=global] - The global state object
+ * @param {object} action - The global action object
+ * @returns {object} The global state object
+ */
+export default function globalReducer(state = global, action) {
+  if (action.type === TRIGGER_AUTHENTICATED) {
```

Comment:
#### Nitpick
- `TRIGGER_AUTHENTICATED` why not `AUTHENTICATION_SUCCESS`/`AUTHENTICATION_FAILURE`

Reply:
Thanks for pointing that, I think `AUTHENTICATION_SUCCESS` sounds good. I will consider refactoring.